consoles/video_stream.pm

```diff
@@ -116,7 +118,8 @@ sub connect_remote_video ($self, $url) {
     $self->{ffmpegpid} = open($ffmpeg, '-|', @$cmd)
       or die "Failed to start ffmpeg for video stream at $url";
     # make the pipe size large enough to hold full frame and a bit
-    fcntl($ffmpeg, Fcntl::F_SETPIPE_SZ, 4194304);
+    my $frame_size = 1680 * 1050 * 3 + 20;    # resolution * 3 bytes/pix + header
```

Comment:
The resolution may not be a fixed value, no?

Reply:
it doesn't seem to have to match exactly - just to be large enough.

Reply:
pleaes don't use magic numbers though

Reply:
I thought I was reducing them with this PR by at least providing a proper calculation with a proper comment.
How would you do it instead?

Reply:
```suggestion
    my $frame_size = $bmwqemu::vars{VIDEO_STREAM_PIPE_BUFFER_SIZE} // (1680 * 1050 * 3 + 20);    # resolution * 3 bytes/pix + header
```

Unfortunately it doesn't make sense to use the `XRES`/`YRES` variables here because supposedly the resolution might not match that (which is another problem to be resolved but likely out of scope for now).